atlasdb-cassandra-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientPoolTest.java

```diff
@@ -128,4 +135,120 @@ public void testIsRetriableException() {
         Assert.assertTrue(CassandraClientPool.isRetriableException(new TTransportException(new SocketTimeoutException())));
     }
 
+    @Test
+    public void testWeightedHostsWithUniformActivity() {
```

Comment:
how do these InetSocketAddress( low number in system reserved range) or InetSocketAddress(0 which means bind any ephemeral) even work